persistence/sql/src/test/java/org/infinispan/persistence/sql/AbstractSQLStoreFunctionalTest.java

```diff
@@ -519,6 +545,23 @@ protected void createTable(String cacheName, String tableName, ConnectionFactory
                "sex VARCHAR(255) NOT NULL, " +
                "name VARCHAR(255), " +
                "PRIMARY KEY (sex))";
+      } else if (upperCaseCacheName.equals("TESTNUMERICCOLUMNS")) {
+         if (DB_TYPE.equals(DatabaseType.SQLITE)) {
+            tableCreation = "CREATE TABLE " + tableName + " (" +
+                  "keycolumn INTEGER, " +
```

Comment:
Can we change INTEGER and NUMERIC(10,0) below to just be a integerType() method like we do with others such as binaryType() etc.

Reply:
Thanks, much better!